src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -150,6 +197,10 @@ export function transform(formConfig, form) {
     // if there is at least one treatment to send
     ...(treatments.length && { treatments }),
     ...(attachments.length && { attachments }),
+    form4142: {
+      limitedConsent: gatherLimitedConsentText(disabilities),
+      providerFacility: transformProviderFacilities(providerFacilities),
```

Comment:
Can you change this to `providerFacilities` to reflect that it's an array?

Reply:
So I can't change the key here. The schema was approved a while back for 4142, the `vets-api` and `vets-json-schema` both have this as `providerFacility`. I will keep in mind the concern of plural variable/property names for lists/arrays, and advise our team.